src/set.js

```diff
@@ -41,10 +44,12 @@ function set(basePath, input, output, terminal) {
         step++
 
         if (step > 3) {
+            let localDate = new Date()
+            //localDate = `${localDate.getFullYear()}-${localDate.getMonth() + 1}-${localDate.getDate()}:${localDate.getHours()}:${localDate.getMinutes()}:${localDate.getSeconds()}`
             if (!info.created)
-                info.created = new Date()
+                info.created = localDate
             else
-                info.modified = new Date()
+                info.modified = localDate
```

Comment:
WARN! this dates are part of the output file, not the terminal, so it should be wrapped with colouring code.